docs/narr/MyProject/development.ini

```diff
@@ -1,6 +1,6 @@
 ###
 # app configuration
-# http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+# http://docs.pylonsproject.org/projects/pyramid/en/1.7-branch/narr/environment.html
```

Comment:
Also I don't plan on updating the scaffolds for every minor or bug-fix release. IIRC, I think I changed it from 1.1-branch to latest at some point, and now I realize that's not good either.

The only time this should change is when we update the scaffolds, which is pretty rare.